src/components/Spacing/Wrapper.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import cx from 'classnames'
+import styles from 'components/Spacing/Wrapper.styl'
+
+const Wrapper = props => {
```

Comment:
Nit: A comment explaining what this comment is about would be appreciated :)

Reply:
Now using NarrowContent from cozy-ui